.github/workflows/test.yml

```diff
@@ -55,12 +57,12 @@ jobs:
         bundler-cache: true
 
     - name: Run sinatra tests
-      continue-on-error: ${{ matrix.allow-failure || false }}
+      continue-on-error: true
```

Comment:
This was to get feedback on all three test suites, should be restored before merge (same for the `continue-on-error` for sinatra-contrib tests)